classes/02class/exercises/c02-network08/README.md

```diff
@@ -6,7 +6,7 @@ Everything is to be done preferable on Sydney region `ap-southeast-2`
 
 #### Look for the network ACL created for your VPC and answer the following:
 
-Your ACL rule inbound and outbound should look like this:
+Your ACL has both inbound and outbound rules as follow:
 
 |Rule#|Type|Protocol|Port Range|Source|Allow/Deny|
 |-|-|-|-|-|-|
```

Comment:
again, sorry for the interruption, for tables it should use 3 or more hyphens eg: |---|---|---|---|---|---| 
reference: https://www.markdownguide.org/extended-syntax

Reply:
GitHub "understands" one hyphen only, but other renderers (like my IDE which is set to strict mode) does not.